mne/conftest.py

```diff
@@ -127,6 +127,7 @@ def pytest_configure(config):
     ignore:.*np\.MachAr.*:DeprecationWarning
     ignore:.*Passing unrecognized arguments to super.*:DeprecationWarning
     always::ResourceWarning
+    ignore:.*Concatenation of Annotations within Epochs.*:RuntimeWarning
```

Comment:
```suggestion
```
+1 for removing this, assuming it doesn't break a ton of tests

Reply:
Removed.